eclass/cmake.eclass

```diff
@@ -495,6 +495,11 @@ cmake_src_configure() {
 				set(CMAKE_FIND_ROOT_PATH_MODE_INCLUDE ONLY)
 			_EOF_
 		fi
+	elif [ "${SYSROOT:-/}" != "/" ] ; then
```

Comment:
```suggestion
	elif [[ ${SYSROOT:-/} != / ]] ; then
```

We tend to use `[[ ... ]]` in bash, and avoid redundant quoting.

Reply:
Done